### PR TITLE
Fix deployment by adding dependencies' environment configuration

### PR DESCRIPTION
**Pull request by sldblog**
## What does this pull request do?

Adds `COMMUNITY_API_URL` and `INTERVENTIONS_SERVICE_URL` to the deployment configuration.

Also removes unused `HMPPS_COOKIE_NAME` and `REDIS_ENABLED`.

## What is the intent behind these changes?

These are required to boot the application up in Kubernetes.

### CODE REVIEW

**Comment by sldblog** [comment]
deploy worked
```
$ helm upgrade hmpps-interventions-ui hmpps-interventions-ui --install --wait --reset-values --timeout 5m --history-max 10 --values values-dev.yaml --set image.tag="2020-12-03.342.1b55882"
Release "hmpps-interventions-ui" has been upgraded. Happy Helming!
```

**Review by lawrence-forooghian — APPROVED** [review]

File: helm_deploy/values-dev.yaml [hunk 1]
```diff
@@ -16,8 +16,8 @@ ingress:
 
 env:
   HMPPS_AUTH_URL: https://sign-in-dev.hmpps.service.justice.gov.uk/auth/
-  HMPPS_COOKIE_NAME: hmpps-session-dev
+  COMMUNITY_API_URL: https://community-api.test.delius.probation.hmpps.dsd.io
```
Comment (tomsmyers) [inline helm_deploy/values-dev.yaml:19]: did we get any clarity on this vs the `secure` version? i looked at my local configs and i generally use the latter.

Comment (sldblog) [reply helm_deploy/values-dev.yaml:19]: No, good point, I still need to do that.

Comment (sldblog) [reply helm_deploy/values-dev.yaml:19]: Other apps use the secure one, so I'll go with that.

Comment (sldblog) [reply helm_deploy/values-dev.yaml]: done in c3977c82679022a50fa56f9c75428f2a94fcd5d2

**Review by tomsmyers — APPROVED** [review]

**Pull request merged**